.github/workflows/publish_npm.yml

```
@@ -0,0 +1,25 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [released]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies
+        run: yarn install --immutable
+      - name: Copy README
+        run: cp README.md packages/upset/README.md
+      - name: Extract tag name
+        id: get_tag
+        run: echo "TAG=${GITHUB_REF#refs/tags/} " >> $GITHUB_OUTPUT
```

Sorry, my suggestion had an extra space that might cause issues
```suggestion
        run: echo "TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_OUTPUT
```

---

Resolved in [bae2b9f](https://github.com/visdesignlab/upset2/pull/361/commits/bae2b9fcb606e3b81d781486a406bc9c2bb1c338)